wandb/apis/public.py

```diff
@@ -1834,7 +1832,7 @@ class Reports(Paginator):
     QUERY = gql(
         """
         query Run($project: String!, $entity: String!, $reportCursor: String,
-            $reportLimit: Int = 50, $viewType: String = "runs", $viewName: String) {
+            $reportLimit: Int!, $viewType: String = "runs", $viewName: String) {
```

Comment:
If you call this `perPage` instead of `reportLimit` you can drop the override method: `update_variables` and use the Pagination class version.

Also, while reportLimit/perPage might indeed by non nullable, that specification isnt consistent with the rest of the file of either:
`$perPage: Int = 50`
or
`$perPage: Int`